test/e2e/framework/pod/resize.go

```diff
@@ -456,6 +456,14 @@ func WaitForPodResizeActuation(ctx context.Context, f *framework.Framework, podC
 func ExpectPodResized(ctx context.Context, f *framework.Framework, resizedPod *v1.Pod, expectedContainers []ResizableContainerInfo) {
 	ginkgo.GinkgoHelper()
 
+	// Cgroup Values are verified by execing into the containers, so we first wait for them to have started.
```

Comment:
I don't think this is the right place to put the wait logic. 2 options:

1. Move it to `WaitForPodResizeActuation`. Maybe just add a condition to that that waits for the pod to be ready? I think that should be sufficient here.
2. Add it to `VerifyPodContainersCgroupValues`, to verify each container is ready as it's used.

I think I prefer option 1?

Reply:
sg! updated this so that it just adds a check to `WaitForPodResizeActuation` to make sure the pod is ready